zulipterminal/ui.py

```diff
@@ -13,6 +13,7 @@
 from zulipterminal.ui_tools.views import (
     LeftColumnView, MiddleColumnView, RightColumnView,
 )
+from zulipterminal.urwid_types import urwid_Size
```

Comment:
do we still want to import it here, even if we don't use it yet?

Reply:
I did notice this, but if we can fix/check the usage near the FIXME, then this wouldn't be an issue, but I'm not sure if @amanagr is too busy to look right now. If so, I'll merge it quicker and we can address that remaining type difference later.

From memory the type hinting is not always more stringent, but just more closely matching the expectations described in the urwid documentation. That might inspire some review of the behavior of these methods.